src/com/itschool/session6/homework/challenge03/Challenge03.java

```diff
@@ -0,0 +1,21 @@
+package com.itschool.session6.homework.challenge03;
+
+import java.util.Scanner;
+
+public class Challenge03 {
+
+    public static void main(String[] args) {
+        checkAge();
+
```

Comment:
unnecessary empty line here.. maybe it makes more sense to add an empty line before `main` and `checkAge` methods.

Reply:
solved

Reply:
solved